client/src/components/utils_sasCode.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
```

Comment:
I didn't realize we already have https://github.com/sassoftware/vscode-sas-extension/blob/main/client/src/components/utils.ts . Can we add this function there? 

Or, if we want to keep these files more tiny/focused, then maybe we create `utils/throttle.ts` (to replace `utils.ts`) and `utils/sasCode.ts`?

Reply:
Either way is fine with me. I'll move it into existing utils.ts then, since it isn't very big currently. Can always refactor later if it does get too large.